apps/showcase/src/components/showcase/rules-engine/rules-engine-pres.template.html

```diff
@@ -1,7 +1,7 @@
 <div class="card my-3">
   <div class="row g-0">
     <div class="col-4 bg-body-tertiary d-flex align-items-center justify-content-center">
-      <img [src]="'otter.svg' | o3rDynamicContent" class="img-fluid rounded-start" aria-hidden="true">
+      <img [src]="'otter.svg' | o3rDynamicContent" class="img-fluid rounded-start visual-testing-ignore" aria-hidden="true">
```

Comment:
Not an blocker but why ignoring this part?
In contrary it is not an image that you would potentially want to visual test?

Reply:
this image changes depending of the time of the year (summer vs not summer)